bin/asset-size-tracking/src/library.js

```diff
@@ -8,11 +8,13 @@ const TablePads = {
 const BROTLI_OPTIONS = {
   params: {
     [zlib.constants.BROTLI_PARAM_MODE]: zlib.constants.BROTLI_MODE_TEXT,
+    // brotli currently defaults to 11 but lets be explicit
+    [zlib.constants.BROTLI_PARAM_QUALITY]: zlib.constants.BROTLI_MAX_QUALITY,
   },
 };
 
 function getCompressedSize(code) {
-  return Buffer.byteLength(zlib.brotliCompressSync(code, BROTLI_OPTIONS));
+  return byteCount(zlib.brotliCompressSync(code, BROTLI_OPTIONS));
```

Comment:
I don't think we should do the compression inside a function named getCompressedSize. Maybe create a separate function to actually compress.